library/alloc/src/vec.rs

```diff
@@ -867,7 +860,7 @@ impl<T> Vec<T> {
     ///
     /// [`truncate`]: #method.truncate
     /// [`resize`]: #method.resize
```

Comment:
These are implemented on `Vec`, not on slices.
```suggestion
    /// [`truncate`]: Vec::truncate
    /// [`resize`]: Vec::resize
```